namespaces/systemd.yml

```diff
@@ -0,0 +1,200 @@
+
+namespace:
+  name: systemd
+  type: group
+  description: >
+    Contains common fields specific to systemd journal
```

Comment:
Add a link to the journald docs for the field names, and then in the description of each field name below put the full name from the journald docs to show the mapping between them.